File: server/karoodaTestToolServer/Models/Result.cs
using karoodaTestToolServer.DAL;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace karoodaTestToolServer.Models {
    public class Result : IToDynamicObject {
        public int id;
        public int belongsToTestcase;
        public int startDate;
        public bool successfull;
        public string result = "";

        public Result() {

        }

        public Result(int belongsToTestcase, int startDate,bool successfull) {
            this.startDate = startDate;
            this.belongsToTestcase = belongsToTestcase;
            this.successfull = successfull;
        }

        public object ToObject() {
            return new {startDate, result, belongsToTestcase, successfull };
        }

        public object ToObjectWithID() {
            return new {id, startDate, result, belongsToTestcase, successfull };
        }
    }
}

File: client/src/models/result.ts
class Result{
    id:number
    belongsToTestcase:number
    startDate:number
    result:string
    successfull:boolean


    constructor(belongsToTestcase: number,startDate: number,result: string,succeeded: boolean){
        this.belongsToTestcase = belongsToTestcase
        this.startDate = startDate
        this.result = result
        this.successfull = succeeded
    }
}